frontend/apps/reader/modules/readerview.lua

```diff
@@ -43,9 +43,9 @@ local ReaderView = OverlapGroup:new{
     page_states = {},
     scroll_mode = "vertical",
     page_gap = {
-        width = Screen:scaleBySize(8),
-        height = Screen:scaleBySize(8),
-        color = Blitbuffer.COLOR_GREY,
+        width = Screen:scaleBySize(DPAGE_GAP_WIDTH),
```

Comment:
Could we do this with `G_reader_settings:readSetting` instead? We are in the processing of migrating everything in global DEFAULTS to `reader.settings.lua` to reduce number of global variables. Then you can you move the comments into the code. I am thinking of writing a custom parser to parse all these tweakable settings to automatically generate a user manual.